lib/checkPullRequestLabels.js

```diff
@@ -285,6 +285,15 @@ module.exports.checkChangelogLabel = async function(context) {
       'guidance. Thanks!',
   });
   await context.github.issues.createComment(stateCommentParams);
+
+  // We are already adding the default changelog label for non collaborators,
```

Comment:
This case will only be active for collaborators since we are ...

Reply:
Done.